src/app/app-routing.module.ts

```diff
@@ -101,7 +101,7 @@ const routes: Routes = [
 
 @NgModule({
   imports: [
-    RouterModule.forRoot(routes, { preloadingStrategy: NoPreloading }),
+    RouterModule.forRoot(routes, { preloadingStrategy: NoPreloading, relativeLinkResolution: 'legacy' }),
```

Comment:
Check to see if this can be removed, explanation here https://stackoverflow.com/questions/65734731/angular-relativelinkresolution-differences-between-legacy-and-corrected